cloud/src/common/configbase.cpp

```diff
@@ -224,6 +228,38 @@ bool Properties::load(const char* conf_file, bool must_exist) {
     return true;
 }
 
+// dump props to conf file
+bool Properties::dump(const std::string& conffile) {
```

Comment:
Describe the behavior of this function, how does the properties are dumped and what happen to the conffile, overwrite or append?